setup.py

```diff
@@ -2,15 +2,15 @@
 
 setup(
     name="applepay",
-    version="0.3.1",
+    version="0.4.0",
```

Comment:
I think bumping this minor version is reasonable getting to your point in the description. It's definitely big enough to warrant it, but smaller than a major update.